language/if_spec.rb

```diff
@@ -1,6 +1,25 @@
 require File.expand_path('../../spec_helper', __FILE__)
 
 describe "The if expression" do
+  ruby_version_is '2.4' do
+    describe "multiple assignments in conditional expression" do
+      before(:each) { ScratchPad.record([]) }
+      after(:each)  { ScratchPad.clear }
+
+      it 'with non-nil values' do
+        collector = proc { |i| ScratchPad << i }
+        eval "if (a, b = [1, 2]); collector[123]; end"
```

Comment:
Why not just using `ScratchPad << 123` here? (same below)
ScratchPad is MSpec's collector :)